src/main/java/by/yahor_kulesh/dao/TicketDAO.java

```diff
@@ -0,0 +1,119 @@
+package by.yahor_kulesh.dao;
+
+import by.yahor_kulesh.model.tickets.BusTicket;
+import by.yahor_kulesh.model.tickets.ConcertTicket;
+import by.yahor_kulesh.model.tickets.Ticket;
+import by.yahor_kulesh.utils.ObjectArray;
+
+import java.sql.*;
```

Comment:
avoid squashed imports

Reply:
IDEA automatically made it - solved

Reply:
yes, but you can configure it in settings